### PR TITLE
Roadmap last updated

### PR DESCRIPTION
**Pull request by asantos3026**
## Launch Checklist

 - [X] briefly describe the changes in this PR
- removed hard coded last updated date in the roadmap's page
- added an updated_at key and configured yml file
- accessed value of updated_at key in the roadmap page
- iterated through items in the roadmap yml to show in the roadmap's page

 - [ ] write tests for all new functionality

 - [ ] post benchmark scores

### CODE REVIEW

**Comment by mollymerp** [comment]
✨ awesome! thanks @asantos3026! You figured it out before I had a chance to respond to the ticket 😸  I'd say this is definitely an improvement on what we have currently. 

If you could `git rebase -i` and only keep the commits relating to this PR on this branch (`drop` the ones from the stylelint PR) that would be great.

closes #4464

**Comment by asantos3026** [comment]
@mollymerp I removed the commits that were related to stylelint 👍

File: package.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
     "test-unit": "tap --reporter dot --no-coverage test/unit",
     "test-render": "node test/render.test.js",
     "test-query": "node test/query.test.js",
-    "test-flow": "flow .",
+    "test-flow": "flow . ",
```
Comment (mollymerp) [inline package.json]: nit – looks like an accidental space was added here 😊

File: .stylelintrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mollymerp) [inline .stylelintrc]: this file should be taken out of this PR – it should already be in `master` I think.

**Comment by mollymerp** [comment]
Thanks for the update, and sorry for the delay in reviewing here @asantos3026! Just a couple small things before this is ready to go 👍

**Comment by mollymerp** [comment]
Including these changes over in #4749 to get this merged! ty @asantos3026 !!

**Pull request closed without merging**